content/2021/04-09-token-exchange-optional-parameter-added.md

```diff
@@ -0,0 +1,32 @@
+---
+applied_at: '2021-04-09'
+applies_to:
+  - api
+is_impactful: false
+is_new_feature: true
+collapse: true
+show_excerpt: true
+release_source_url: ''
+---
+
+# Token Exchange Optional Parameter Added
```

Comment:
Recommend moving away from "token exchange" as we typically refer to the process as token downscoping in the documentation, and provide a bit more info on the context of the change. I'd probably recommend something like "New option for downscoping tokens using shared links".